packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -0,0 +1,67 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    borderWidth,
+    controlHeight,
+    fillHoverColor,
+    mediumDelay,
+    standardPadding,
+    tableRowBorderColor
+} from '../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('flex')}
+
+    :host {
+        align-items: center;
+        height: calc(${controlHeight} + 2 * ${borderWidth});
+        border-top: calc(2 * ${borderWidth}) solid ${tableRowBorderColor};
+        padding-left: calc(
+            ${standardPadding} * 2 *
+                var(--ni-private-table-group-row-indent-level)
+        );
+    }
+
+    :host(:hover) {
+        background: ${fillHoverColor};
+    }
+
+    :host([expanded='true']) .expander-icon,
+    :host([expanded]) .expander-icon {
+        transform: rotate(90deg);
+    }
+
+    :host([expanded='false']) .expander-icon {
+        transform: rotate(0deg);
+    }
+
+    :host([expanded='true']) .animating,
+    :host([expanded='false']) .animating {
+        transition: ${mediumDelay} ease-in-out;
```

Comment:
I'm using `mediumDelay` (0.15s) here to a) leverage a token instead of a hard-coded value, and to provide an intermediate animation time between our hover actions (`smallDelay`), and the time for a drawer to open (`largeDelay`).